fe/fe-core/src/main/java/org/apache/doris/nereids/stats/FilterEstimation.java

```diff
@@ -222,9 +221,12 @@ private Statistics estimateEqualTo(ComparisonPredicate cp, ColumnStatistic stats
         double val = statsForRight.maxValue;
         if (val > statsForLeft.maxValue || val < statsForLeft.minValue) {
             selectivity = 0.0;
+        } else if (statsForRight.isUnKnown) {
+            selectivity = 0.0;
```

Comment:
right is a literal, why it could be unknow

Reply:
ok, will remove it.